### DISCUSSION TITLE
Obsidian not making I.D numbers & freezing.

### DISCUSSION THREAD

**Discussion by Xtuhfurr** [ObsidianToAnki/Obsidian_to_Anki#612] [General]
I've gone through this discussion board and have read all of the recent ish posts about this happening to other people and tried all of the possible things to try and sort it out but nothing has worked. 

I have: 
disabled obsidian to anki plug in 
deleted and reinstalled it 
re entered the note setting "basic" code ((?:[^\n][\n]?)+) #flashcard ?\n*((?:\n(?:^.{1,3}$|^.{4}(?<!<!--).*))+)
tried to change line 30603 in the obsidian file 
ctrl - shift - I and watched to see what file it got stuck on and deleted it 

I'm kind of stuck now. 
When trying to sync it'll either say everything is done and good or it'll freeze. It picks and chooses between those options! 

It'll create a "TARGET DECK" within Anki but won't upload any of the flashcards and obsidian isn't creating any of the I.Ds like it use to. 

any help would be appreciated!

**Comment by Xtuhfurr** [comment]
Not joking, after a few days of researching this problem and finally posting on this board, it all of a sudden just began working again lol. 

I don't know what's going on.

**Comment by envico801** [comment]
Hi @Xtuhfurr, there are some times that the plugin can get “frozen” particularly if you want to add a lot of cards at once. For example, in my case, adding around >=300 cards takes about 15 minutes to be sent to anki.

Another cause may be that your anki vault is too big and the plugin has to scan each file for changes. To solve this you can either create a separate vault for testing or use expressions to tell the plugin to check only certain directories. For example save all your cards in the folder `./my-cards`.

Then if the plugin doesn't detect the changes in a file you can try to change or add some letter or something and try to sync again. That should make it sync correctly.
